contracts/dao/FeiDAO.sol

```diff
@@ -78,6 +91,33 @@ contract FeiDAO is
         emit ProposalThresholdUpdated(oldProposalThreshold, newProposalThreshold);
     }
 
+    /// @notice one-time option to roll back the DAO to old GovernorAlpha
+    /// @dev admin-only, and expires after the deadline. This function is here as a fallback in case something goes wrong.
+    function __rollback(uint256 eta) external {
+        require(msg.sender == _admin, "FeiDAO: caller not admin");
+        // Deleting admin prevents multiple triggers of this function
+        _admin = address(0);
+
+        require(eta <= ROLLBACK_DEADLINE, "FeiDAO: rollback expired");
```

Comment:
We should also require that eta < now. I know this is checked further along when the timelock checks this too, but checking here makes it simpler to analyze.

Reply:
The timelock will revert if eta is < now. I'd lean against adding it but if you feel strongly its not much different

Reply:
Fine by me